velox/experimental/codegen/ast/ConstantExpressions.h

```diff
@@ -141,7 +141,7 @@ class ConstantExpression final : public ASTNode {
  private:
   /// Child expression
   const std::optional<T> value_;
-}; // namespace codegen
```

Comment:
This semi-colon is needed, but the comment is wrong and needs to be removed.

Reply:
Fixed, thanks.